README.md

```diff
@@ -260,3 +260,23 @@ example uses JavaScript's `btoa` and `atob` functions for string
 conversion. For dealing with files directly, using the
 [HTML5 File API](https://developer.mozilla.org/en-US/docs/Web/API/FileReader.readAsDataURL)
 would be a good option.
+
+## SSH key encryption
```

Comment:
Good job with this description. I like how concise it is while still showing the user what to do